gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,6 @@
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-bin.zip
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
```

Comment:
Have you considered `gradle-8.6.bin` instead?

Reply:
Updated to 8.6 and it comes with new warnings.
Any help to fix them will be appreciated because most of them need a lot of deep gradle knowledge.